HelpSource/Guides/UserFAQ.schelp

```diff
@@ -0,0 +1,418 @@
+title:: User FAQ
+summary:: Some FAQ and common errors
+categories:: FAQ
+
+- link::#ERROR Primitive BasicNew failed Index not an Integer#ERROR: Primitive 'BasicNew' failed. Index not an integer::
+
+- link::#Language (client) Issues#Language (client) Issues::
+
+- link::#SynthDef Issues#SynthDef Issues::
+
+- link::#Server Issues#Server Issues::
+
+- link::#Other Issues#Other Issues::
```

Comment:
I don't think we need a manual table of contents like this. There's one generated automatically available in the top right corner of the help browser and this one doesn't look that great anyway. IMO this could be simply removed. Also, you probably don't need `anchor`s later on when this is not present.

Reply:
@dyfer I'll remove the table but anchors are needed because we link directly to those sections in various helpfiles

Reply:
Subsections get a link automatically, AFAIR